README.md

```diff
@@ -1,5 +1,24 @@
 # NPS main theme plugin
 
+## Installing MainTheme dependencies
+
+To use the MainTheme you need to install npm dependencies.
+Go to Theme folder:
+```
+cd custom\plugins\MainTheme
+```
+
+Install npm dependencies
+```$xslt
+npm install
+```
+
+Run script that setups GLightbox
+```$xslt
+npm run glightbox-setup
```

Comment:
Could we perhaps put this, together with `npm install`, in some custom npm script? Or use npm hooks to run this after installation?